### PR TITLE
Book a demo: copy update

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

The current copy was created aligned to the previous messaging, this PR updates it to better align it with the current one.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/book-demo.njk [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: layouts/mql-contact.njk
 title: Book a demo
-subtitle: Discover the power of FlowFuse for Node-RED projects!
-description: Book a software demo to better understand how to supercharge your workflow with high availability, enhanced data manipulation, and simplified automation. Book now for efficient Node-RED deployments!
+subtitle: Experience the fastest way to build and operate industrial data solutions
```
Comment (Hasmin-AC) [inline src/book-demo.njk:4]:
```suggestion
subtitle: Get customized advice and recommendations to ensure you get the most out of FlowFuse.
```

File: src/book-demo.njk [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+description: See how you can streamline operations, enhance collaboration, and secure your Node-RED solutions for seamless IT/OT integration. Ready to accelerate your digital transformation? Book your demo today!
```
Comment (Hasmin-AC) [inline src/book-demo.njk:5]: Tap into our expertise to optimize operations, improve collaboration, and ensure compliance for your Node-RED solutions without any pressure or commitment.

**Review by Hasmin-AC — APPROVED** [review]

**Comment by Hasmin-AC** [comment]
suggest text that focuses more what you get if you book a call.

File: src/book-demo.njk (hunk 1 above)
Comment (Yndira-E) [inline src/book-demo.njk:4]:
```suggestion
subtitle: Get customised advice and recommendations to ensure you get the most out of FlowFuse.
```

File: src/book-demo.njk (hunk 2 above)
Comment (Hasmin-AC) [inline src/book-demo.njk:5]:
```suggestion
description: Tap into our expertise to optimize operations, improve collaboration, and ensure compliance for your Node-RED solutions without any pressure or commitment.
```

**Review by Hasmin-AC — APPROVED** [review]

File: src/book-demo.njk [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: layouts/mql-contact.njk
 title: Book a demo
-subtitle: Discover the power of FlowFuse for Node-RED projects!
-description: Book a software demo to better understand how to supercharge your workflow with high availability, enhanced data manipulation, and simplified automation. Book now for efficient Node-RED deployments!
+subtitle: Get customised advice and recommendations to ensure you get the most out of FlowFuse.
+description: See how you can streamline operations, enhance collaboration, and secure your Node-RED solutions for seamless IT/OT integration. Ready to accelerate your digital transformation? Book your demo today!
```
Comment (Yndira-E) [inline src/book-demo.njk:5]:
```suggestion
description: Tap into our expertise to optimise operations, improve collaboration, and ensure compliance for your Node-RED solutions without any pressure or commitment.
```

**Comment by Hasmin-AC** [comment]
@Yndira-E Can we get the description to span the same length as the sub title?

**Comment by Yndira-E** [comment]
> Can we get the description to span the same length as the sub title?

Done! (I was already on it 😉)

**Pull request merged**